src/coverlet.collector/Utilities/TestPlatformEqtTrace.cs

```diff
@@ -8,17 +8,17 @@ namespace Coverlet.Collector.Utilities
     /// <summary>
     /// Test platform eqttrace
     /// </summary>
-    internal class TestPlatformEqtTrace
+    internal static class TestPlatformEqtTrace
```

Comment:
Updated the type to be `static` as all members could be `static`.  If you prefer an instance with the idea of one day have a different instance for tests/prod then let me know I can revert. We could also wait for such need and refactor and that time.

Reply:
Yep pls revert I want to able to test/inject everything